pkg/securitypolicy/open_door.rego

```diff
@@ -1,14 +1,14 @@
 package policy
 
-api_svn := "0.10.0"
+api_svn := "@@API_SVN@@"
```

Comment:
rather than an ad-hoc replacemnt marker, would go [templates](https://pkg.go.dev/text/template) work? 
I think the only issue would be if `{{` and `}}` are valid rego syntax and would need to be escaped if present...

Reply:
I was unaware of go templates. Let me give it a try, I would prefer not to use ad hoc replacement markers if possible.

Reply:
So I looked into converting the current markers to go templates, but the fact that templates work with streams makes it quite a big change. I'll add this to the nit list so it can be done as a standalone change.